aexpect/remote.py

```diff
@@ -288,17 +290,19 @@ def handle_prompts(session, username, password, prompt=PROMPT_LINUX,
                     else:
                         msg = "Got username prompt after password prompt"
                     raise LoginAuthenticationError(msg, text)
-            elif match == 5:  # "Connection closed"
+            elif match == 5:   # "Connection closed"
```

Comment:
I see some of these comments having three spaces before the inline comment while others have two, I am not sure if this is intentional.

Reply:
I kept the change in original avocado-vt PR. 
I think it's for keeping them at same column to look tidy?
Anyway, I googled the PEP8, it only says 'Inline comments should be separated by at least two spaces from the statement.' IMO, it's fine. But I could change it back if you think it's a style in aexpect.
Thanks.

Reply:
No particular style and I thought the same thing about at least two spaces. I was only wondering because some of the other inline comments used two spaces (the minimum) and these used three even though they are neighboring lines. But this is just a minor thing and not too important, let's keep the VT formatting for now and unify later.

Reply:
Well ideally a commit should either change style, refactor or implement new features. It should not change multiple things to simplify reviews as well as reverts, but let's keep it for now.

Reply:
We should use `python-black` to mass format it and then all the discussions become moot.